### PR TITLE
Fixes #1785 Inconsistent curve naming

### LINKED ISSUE #1785: 11.1 Inconsistent naming in time profile plots and GOF plots
@Yuri05 commented on [Fri Nov 11 2022](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2420)

E.g. simulated Arterial Blood/Plasma curve is displayed like this in a time profile plot:
grafik

And in Predicted vs. Observed and Residuals vs. time like this:
grafik

---

@rwmcintosh commented on [Mon Nov 14 2022](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2420#issuecomment-1313874931)

Should be checked if the same behavior is observed for PI and Simulation

### PR DESCRIPTION
**Pull request by rwmcintosh**

### CODE REVIEW

File: src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationPredictedVsObservedChartPresenter.cs
Comment (rwmcintosh) [inline src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationPredictedVsObservedChartPresenter.cs:80]: Behavior was always to just use the `PathAsString` since PI but we have this chart method called `NameForColumn` available

File: src/OSPSuite.Presentation/Presenters/SimulationVsObservedDataChartPresenter.cs
Comment (msevestre) [reply src/OSPSuite.Presentation/Presenters/SimulationVsObservedDataChartPresenter.cs:76]: Can you leave a comment why we remove the simulation Name here (in fact just the comment of the PR
`In the case of these charts that are single simulation, we just need to override the default implementation to not include the simulation name.` then you can merge

Comment (rwmcintosh) [reply src/OSPSuite.Presentation/Presenters/SimulationVsObservedDataChartPresenter.cs:76]: Just put the approval on it and don't merge. I'll merge when the build finishes in appveyor

**Review by msevestre — APPROVED** [review]

**Pull request merged**
